engine/src/canvas.mlc

```diff
@@ -3729,6 +3729,10 @@ Description:
 
 >*Note:* The layout bounds is used to position blocks of text relative to each other, so the text may overlap these bounds.
 
+The `layout` is optional and has no effect on the return value, but may be used to emphasize the difference between `the layout bounds` and `the image bounds`.
```

Comment:
The point here is not that it's optional, but that it has no affect on the result.  If this isn't said then the developer (or me yesterday!) is left wondering what it does and whether they need it or not.